src/Veldrid.VirtualReality/Veldrid.VirtualReality.csproj

```diff
@@ -14,21 +14,20 @@
     <PackageReference Include="Veldrid.SPIRV" Version="1.0.2" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(UseStableVersions)' == 'true'" >
+  <ItemGroup Condition="'$(UseStableVersions)' == 'true'">
     <PackageReference Include="Veldrid" Version="4.5.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(UseStableVersions)' != 'true'" >
+  <ItemGroup Condition="'$(UseStableVersions)' != 'true'">
     <ProjectReference Include="..\Veldrid\Veldrid.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="native\win-x64\" />
```

Comment:
I don't think this works as expected. I think that a `Folder` property by itself is only going to make it show up in VS / other IDEs, but I don't think any of the binaries in it will be copied to the output directory.

For that, you need the `Content Include=""` and `CopyToOutputDirectory` snippets that were removed below. I was expecting to see the same kind of logic that already existed for `win-x64\openvr_api.dll` to be duplicated for `win-x86\openvr_api.dll`.